source/images/buyers-guide/icon-close.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
you'll want to run this through svgo, because this is probably <300byte after svg optimization